problems/[230619] Top K Frequent Words.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/top-k-frequent-words
+ *
+ * @param {string[]} words
+ * @param {number} k
+ * @return {string[]}
+ */
+const topKFrequent = function (words, k) {
+  const wordFrequent = {};
+
+  words.forEach((word) => {
+    wordFrequent[word] ? (wordFrequent[word] += 1) : (wordFrequent[word] = 1);
+  });
+
+  const wordFrequentArray = Object.entries(wordFrequent);
+
+  wordFrequentArray
+    .sort((a, b) => {
+      return b[1] - a[1];
+    })
+    .sort((a, b) => {
+      if (a[1] === b[1]) {
+        return a[0] > b[0] ? 1 : -1;
+      }
+    });
```

Comment:
```suggestion
    .sort((a, b) => {
      if (a[1] !== b[1]) {
        return b[1] - a[1];
      } else {
        return a[0] > b[0] ? 1 : -1;
      }
    })
```

sort 하나로 줄이면 시간복잡도 면에서 더 개선할 수 있을 것 같아용